rl_zoo3/tracking/wandb.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import sys
+from typing import Dict
+
+import wandb
+from stable_baselines3.common.logger import Logger, HumanOutputFormat
+
+from rl_zoo3.tracking.tracking_backend import TrackingBackend
+
+
+class WandbBackend(TrackingBackend):
+    def __init__(self):
+        super().__init__()
+        self.run = None
+
+    @classmethod
+    def argparse_add_arguments(cls, parser: argparse.ArgumentParser) -> None:
+        parser.add_argument("--wandb-entity", type=str, default=None, help="the entity (team) of wandb's project")
+        parser.add_argument("--wandb-project-name", type=str, default="sb3", help="the wandb's project name")
+        parser.add_argument(
+            "-tags", "--wandb-tags", type=str, default=[], nargs="+", help="Tags for wandb run, e.g.: -tags optimized pr-123"
+        )
+
+    @classmethod
+    def argparse_del_arguments(cls, parsed_args) -> None:
+        del parsed_args.wandb_entity
+        del parsed_args.wandb_project_name
+        del parsed_args.wandb_tags
+
+    def _setup_tracking(self, args) -> None:
+        assert "wandb_entity" in vars(args), "Missing W&B entity (--wandb-entity)."
+
+        tags = set([f"{k}-{v}" for k, v in self.get_tracking_commit_hashes().items()])
+        tags.update([f"{k}-{v}" for k, v in self.get_version_tags().items()])
+
+        self.run = wandb.init(
+            name=args.run_name,
+            project=args.wandb_project_name,
+            entity=args.wandb_entity,
+            tags=tags,
+            config=vars(args),
+            sync_tensorboard=True,  # auto-upload sb3's tensorboard metrics
+            monitor_gym=True,  # auto-upload the videos of agents playing the game
+            save_code=True,  # optional
+        )
+
+    def _finish_tracking(self) -> None:
+        pass
```

Comment:
is this a correct way to finish tracking wandb?

Reply:
Implementation of `wandb` has been taken 1:1 from the orginal rl_zoo3 repository. Should we do more than that?